libcst/_nodes/tests/test_assign.py

```diff
@@ -362,3 +408,26 @@ class AugAssignTest(CSTNodeTest):
     )
     def test_valid(self, **kwargs: Any) -> None:
         self.validate_node(**kwargs)
+
+    @data_provider(
+        (
+            {
+                "get_node": (
+                    lambda: cst.AugAssign(
+                        target=cst.BinaryOperation(
```

Comment:
```suggestion
                        # pyre-fixme: Incompatible parameter type [6]
                        target=cst.BinaryOperation(
```

Reply:
> Hmmm not sure what pyre is complaining about. I tried to reproduce locally but I get tons of errors on master so I must have something configured wrong.
> 
> Edit:
> Possibly my intentional construction of ill-typed nodes? Is there a way to get pyre to skip something?

Pyre did the right thing to complain about the target node are not of `BaseAssignTargetExpression`.
Yes, we can ignore the pyre errors in the test cases as @zsol suggested.
Instead of `pyre-fixme`, use `pyre-ignore` is better because we don't plan to fix them.